nuxwdog.spec

```diff
@@ -133,6 +135,9 @@ rm -rf %{buildroot}
 %{_jnidir}/*
 
 %changelog
+* Mon Jul 23 2018 Dinesh Prasanth M K <dmoluguw@redhat.com> - 1.0.4-2
+- Resolves: 1605267 - add gcc-c++ and perl-devel deps for F29+
```

Comment:
Since 1.0.4-1 was never built in Koji, this line can be merged into 1.0.4-1 change log, then update the date. To avoid conflicting authors we usually put Dogtag PKI Team (pki-devel@redhat.com) instead of individual names.

Reply:
See also https://fedoraproject.org/wiki/Packaging:Guidelines#Updating_and_replacing_the_existing_date_line

Reply:
For PKI we don't actually store the changelog in upstream spec template:
https://github.com/dogtagpki/pki/blob/master/specs/pki.spec.in

We only store the changelogs in the actual spec file:
https://src.fedoraproject.org/rpms/pki-core/blob/master/f/pki-core.spec